projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -46,3 +46,25 @@ export function jobRunRequest() {
             );
         });
 }
+
+ /**
+    * Sent a POST request to the server to delete a data job.
+    * The information about the data job is retrieved from sessionStorage and sent as JSON.
+    */
+  export function deleteRunRequest() {
+    let dataToSend = {
+        jobName: sessionStorage.getItem("delete-job-name"),
+        jobTeam: sessionStorage.getItem("delete-job-team"),
+        restApiUrl: sessionStorage.getItem("delete-job-rest-api-url")
+    };
+    requestAPI<any>('delete', {
```

Comment:
I would imagine names match | link the control service API, or the class doc includes a link to the entire api

Reply:
sorry, I cannot really understand what  you mean by that
the real connection is done in handlers.ts, here I just add the endpoint

Reply:
I will merge it like that for now, we can discuss this while creating the other components